site2/docs/tiered-storage-gcs.md

```diff
@@ -0,0 +1,275 @@
+   
+    ```conf
+    gcsManagedLedgerOffloadServiceAccountKeyFile="/Users/user-name/Downloads/project-804d5e6a6f33.json"
+    ```
+
+    > #### Tip
+    >
+    > - For more information about how to create `gcsManagedLedgerOffloadServiceAccountKeyFile`, see [here](https://support.google.com/googleapi/answer/6158849).
+    >
+    > - For more information about Google Cloud IAM, see [here](https://cloud.google.com/storage/docs/access-control/iam).
+
+#### Size of block read/write
+
+You can configure the size of a request sent to or read from GCS in the configuration file `broker.conf`. 
+
+Configuration|Description
+|---|---
+`gcsManagedLedgerOffloadReadBufferSizeInBytes`|Block size for each individual read when reading back data from GCS.<br><br>The **default** value is 1 MB.
+`gcsManagedLedgerOffloadMaxBlockSizeInBytes`|Maximum size of a "part" sent during a multipart upload to GCS. <br><br>It **can not** be smaller than 5 MB. <br><br>The **default** value is 64 MB.
+
+### Configure GCS offloader to run automatically
+
+Namespace policy can be configured to offload data automatically once a threshold is reached. The threshold is based on the size of data that a topic has stored on a Pulsar cluster. Once the topic reaches the threshold, an offload operation is triggered automatically. 
+
+Threshold value|Action
+|---|---
+> 0 | It triggers the offloading operation if the topic storage reaches its threshold.
+= 0|It causes a broker to offload data as soon as possible.
+< 0 |It disables automatic offloading operation.
+
+Automatic offloading runs when a new segment is added to a topic log. If you set the threshold on a namespace, but few messages are being produced to the topic, offload does not work until the current segment is full.
+
+You can configure the threshold size using CLI tools, such as [pulsarctl](https://streamnative.io/docs/v1.0.0/manage-and-monitor/pulsarctl/overview/) or pulsar-admin.
+
+The offload configurations in `broker.conf` and `standalone.conf` are used for the namespaces that do not have namespace level offload policies. Each namespace can have its own offload policy. If you want to set offload policy for each namespace, use the command [`pulsar-admin namespaces set-offload-policies options`](http://pulsar.apache.org/tools/pulsar-admin/2.6.0-SNAPSHOT/#-em-set-offload-policies-em-) command.
+
+#### Example
+
+This example sets the GCS offloader threshold size to 10 MB using pulsarctl.
+
+```bash
+bin/pulsarctl namespaces set-offload-threshold --size 10M my-tenant/my-namespace
+```
+
+> #### Tip
+>
+> For more information about the `pulsarctl namespaces set-offload-threshold options` command, including flags, descriptions, default values, and shorthands, see [here](https://streamnative.io/docs/pulsarctl/v0.4.0/#-em-set-offload-threshold-em-). 
+
+### Configure GCS offloader to run manually
+
+For individual topics, you can trigger GCS offloader manually using one of the following methods:
+
+- Use REST endpoint 
```

Comment:
```suggestion
- Use REST endpoint.
```

Reply:
This is not a complete sentence which has a subject and predicate. 

IBM style guide:
Do not use a period after a phrase and use a period after a sentence.
![image](https://user-images.githubusercontent.com/50226895/86197914-ffd02980-bb88-11ea-86db-505303534e7e.png)